src/locales/en_US.js

```diff
@@ -541,6 +541,8 @@ const strings = {
   modal_addressexplorer_message: 'Show Address in Block Explorer?',
   modal_addressexplorer_null: 'This currency has no explorer site as of now',
   approximately: 'Estimated',
+  fixed_quote: 'Fixed Quote',
+  estimated_quote: 'Fixed Quote',
   estimated_exchange_rate: 'Estimated Exchange Rate',
```

Comment:
@ahujsak This seems like a typo. One should say `Estimated Quote`

Reply:
Fixed